vehicle/security/ir.model.access.csv

```diff
@@ -0,0 +1,3 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_vehicle_user,access.vehicle.vehicle.user,model_vehicle_vehicle,group_vehicle_user,1,0,1,0
```

Comment:
Write permission is needed by default. If you test with a User role, an error is fired when creating the record (probably constrains method write operation). In general, the user that creates a vehicle, should change it
```suggestion
access_vehicle_user,access.vehicle.vehicle.user,model_vehicle_vehicle,group_vehicle_user,1,1,1,0
```